engine/tester/config_parse.c

```diff
@@ -273,6 +273,9 @@ remove_common_leading_indent(char *str)
     const char *p;
     char       *write_ptr = str;
 
+    if (!(tester_global_context.flags & TESTER_STRIP_INDENT))
```

Comment:
I have doubts that it is a good idea to check it inside the function. IMHO it looks confusing when you call the function with such name, but in fact it does nothing in some cases like this.